tests/chainerx_tests/op_utils.py

```diff
@@ -183,6 +183,13 @@ def _create_test_entry_function(
     # method_name:
     #    The name of the test method name defined in `FunctionTestBase` class.
 
+    if not cls.__name__.startswith('Test'):
```

Comment:
I don't think this code is obvious without having followed the discussions behind this PR. Could you comment why it's needed and maybe also include a TODO since this is a temporary workaround?

Reply:
Added comment.

Reply:
Thanks!